ksm-throttler/cc-ksm-throttler.spec-template

```diff
@@ -0,0 +1,78 @@
+%global PREFIX /usr/
+%global DOMAIN github.com
+%global ORG kata-containers
+%global PROJECT ksm-throttler
+%global IMPORTNAME %{DOMAIN}/%{ORG}/%{PROJECT}
+%global GO_VERSION 1.9.2
```

Comment:
I think this is fine as 1.9.2 is the latest. However, it's worth pointing out that the other golang packages in this repo are using 1.8.3 and the CI is using a different version again. I've raised #206 to capture this issue.

Reply:
Thanks @jodh-intel for raising the issue!

Reply:
Now that #208 has landed, this can be fixed.